### PR TITLE
Update it.json

### PR DESCRIPTION
**Pull request by Lollixzc**
Hi,
Fixed some translations in the locale files. 
I wanted to know how to fix the translation in the web part of the application, because I cannot find the translation file. 
Here is a Screen where I want to fix the translation(Scelgli to Scegli): 
image

Another thing is: under `warn.ssl.renew` you mean that you need to renew fortify SSL certificate right? Because In the translation I assumed it was that. 

Cheers, 
Lollixzc

### CODE REVIEW

**Comment by donskov** [comment]
Hello @Lollixzc. Translation in the web component is fixed. Try:
```
 - @peculiar/fortify-webcomponents-react@0.2.38
 - @peculiar/fortify-webcomponents@0.21.4
 ```

File: locale/it.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
   "question.2key.remove": "Desideri rimuovere %1 dall'elenco dei siti attendibili?",
   "warning.title.oh_no": "Oh no, non ha funzionato!",
   "warn.ssl.install": "È necessario installare il certificato SSL radice di Fortify. Per eseguire questa operazione potrebbe essere necessaria la password di amministratore.",
-  "warn.ssl.renew": "SSL certificate requires renew. Please run the Fortify installer to renew a certificate.",
+  "warn.ssl.renew": " Il certificato SSL di Fortify deve essere rinnovato. Esegui l'installer di Fortify per rinnovare il certificato.",
```
Comment (donskov) [inline locale/it.json:41]: Please remove the empty space before ` Il certificato SSL...`.

**Review by donskov — CHANGES_REQUESTED** [review]

File: locale/it.json (hunk 1 above)
Comment (Lollixzc) [reply locale/it.json]: Resolved the typo

**Review by donskov — APPROVED** [review]

**Comment by donskov** [comment]
@Lollixzc Thanks for your help. Changes are available in [1.8.4](https://github.com/PeculiarVentures/fortify/releases/tag/1.8.4) pre-release version.

**Pull request merged**